genie-agent/src/main/java/com/netflix/genie/agent/cli/UserConsole.java

```diff
@@ -48,33 +48,80 @@
      * consumed by Spring.
      */
     private static final String CONSOLE_LOGGER_NAME = "genie-agent";
-    /**
-     * This string path must match the one present in the log appender configuration consumed by Spring.
-     * It represents the initial location of the agent log file.
-     */
-    private static final String LOG_FILE_PATH = "/tmp/genie-agent-%s.log";
+
     /**
      * This system property is set by Spring.
      */
     private static final String PID_SYSTEM_PROPERTY_NAME = "PID";
     /**
-     * The name of the environment variable that can be supplied to have the temporary log file at a non standard
-     * location.
+     * System property or environment variable to override the full path of the logfile.
+     */
+    private static final String LOG_FILE_PATH_OVERRIDE_PROPERTY_NAME = "GENIE_AGENT_TEMPORARY_LOG_FILE";
+    /**
+     * System property or environment variable to override the directory of the logfile.
+     */
+    private static final String LOG_DIRECTORY_OVERRIDE_PROPERTY_NAME = "GENIE_AGENT_TEMPORARY_LOG_DIRECTORY";
+    /**
+     * System property or environment variable to override the filename of the logfile.
+     */
+    private static final String LOG_FILENAME_OVERRIDE_PROPERTY_NAME = "GENIE_AGENT_TEMPORARY_LOG_FILENAME";
+
```

Comment:
Extra line here relative to your other values. Doesn't matter obviously just pointing it out if you wanted to be consistent.

Reply:
There's an empty line between groups of constants. This one is between property names and default values